packages/grafana-runtime/package.json

```diff
@@ -56,13 +57,15 @@
     "@testing-library/user-event": "14.4.3",
     "@types/angular": "1.8.4",
     "@types/history": "4.7.11",
-    "@types/jest": "28.1.6",
+    "@types/jest": "^29.2.2",
```

Comment:
Can I ask why this was bumped? It would be nice to keep the dependencies used across the app and packages in sync where possible.